### PR TITLE
Display section thumbnails and info for closed hearings

### PR DESCRIPTION
**Pull request by Rikuoja**

### CODE REVIEW

**Comment by Rikuoja** [comment]
Closes #139 #153 #167

**Comment by Rikuoja** [comment]
And #178

File: src/views/Hearing/index.js [hunk 1]
```diff
@@ -128,12 +129,20 @@ class Hearing extends React.Component {
     const hearingAllowsComments = !hearing.closed && (new Date() < new Date(hearing.close_at));
     const onPostVote = this.onVoteComment.bind(this);
     const introSection = find(hearing.sections, (section) => section.type === "introduction");
-    const closureInfoSection = find(hearing.sections, (section) => section.type === "closure-info");
+    let closureInfoSection = find(hearing.sections, (section) => section.type === "closure-info");
```
Comment (akx) [inline src/views/Hearing/index.js]: Would probably be better to let this remain `const` and move the rvalue and the fallback logic (below) into a helper function like `_getClosureInfoSection()`.

**Comment by akx** [comment]
Also, hmm, are we using a `develop` branch instead of feature branches? :open_mouth:

**Comment by Rikuoja** [comment]
Fair point, I'll do features in separate branches in the future.

File: src/views/Hearing/index.js [hunk 2]
```diff
@@ -128,12 +145,12 @@ class Hearing extends React.Component {
     const hearingAllowsComments = !hearing.closed && (new Date() < new Date(hearing.close_at));
     const onPostVote = this.onVoteComment.bind(this);
     const introSection = find(hearing.sections, (section) => section.type === "introduction");
-    const closureInfoSection = find(hearing.sections, (section) => section.type === "closure-info");
+    const closureInfoSection = this.getClosureInfo();
```
Comment (akx) [inline src/views/Hearing/index.js]: Maybe just pass `hearing` in as a param instead of re-deriving it in the helper? :)

Comment (akx) [inline src/views/Hearing/index.js]: (As is done with `getOpenGraphMetaData` etc already)

**Pull request merged**